### PR TITLE
Use sed extended REGEX in server manager config update

### PR DESCRIPTION
**Pull request by RobertRR**
some substitutions are failing.
Resolves multiple similar issues regarding env VAR substitutions.

### CODE REVIEW

File: servermanager.sh
Comment (lnsflive) [inline servermanager.sh:1]: this worked for me

**Comment by jammsen** [comment]
Hello @RobertRR - Can you describe your problem please more so i can try to reprouce and understand the problem better?

**Comment by RobertRR** [comment]
Mostly issues like https://github.com/jammsen/docker-palworld-dedicated-server/issues/66

**Comment by jammsen** [comment]
Hello @RobertRR - Thanks so much for the help in this.
Sadly i have a lot more to change then only this and did this on my develop branch to try if everything works first. Im sorry that i dont use this PR, but i cant add you manually as a contributor 😭

**Comment by jammsen** [comment]
Closed in favor of #79
Again big thanks.

**Pull request closed without merging**